.eslintrc.js

```diff
@@ -26,18 +33,25 @@ module.exports = {
         'jest/no-test-return-statement': 'off',
         // TODO: Re-enable this rule; we can accomodate this even in our test helpers
         'jest/expect-expect': 'off',
+
+        // TODO: Re-enable
+        '@typescript-eslint/naming-convention': 'off',
```

Comment:
Hmm, no lint errors for this rule locally. Maybe we can leave this one enabled?

Reply:
Oh I see, it's disabled in two places (here's the second one: https://github.com/MetaMask/core/pull/1498/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R105)

Reply:
Agreed, I'm definitely a fan of this disabling non-auto-fixable rules approach